javascript/ola/LICENCE

```diff
@@ -1,13 +0,0 @@
-This program is free software; you can redistribute it and/or modify
```

Comment:
@daveol you still haven't resurrected this file, if you put it back, the licence checker can ensure the files are all correctly licensed.